beacon-chain/blockchain/service.go

```diff
@@ -426,6 +438,14 @@ func (c *ChainService) isBlockReadyForProcessing(block *pb.BeaconBlock, beaconSt
 	return nil
 }
 
+func (c *ChainService) checkLastFinalizedEpoch(beaconState *pb.BeaconState) error {
```

Comment:
Can you add a better method name? This doesn’t describe the behavior of the method